src/main/java/uk/gov/ons/ssdc/supporttool/schedule/RowChunkStager.java

```diff
@@ -69,7 +76,9 @@ public JobStatus stageChunk(Job job, String[] headerRow, CSVReader csvReader) {
       jobRepository.saveAndFlush(job);
       jobRowRepository.saveAll(jobRows);
     } catch (IOException e) {
-      throw new RuntimeException(e);
+      log.with("Lines read", csvReader.getRecordsRead())
+          .error("Error staging job row, CSV data is malformed");
```

Comment:
I think this needs the filename and/or the job ID, otherwise we don't know the job/file it's complaining about. It also needs to preserve the original exception message or details in some way, like this we're losing all the actual exception details.

Reply:
Updated this to add more details